tests/MongoDB.Driver.Legacy.Tests/Builders/UpdateBuilderTests.cs

```diff
@@ -838,8 +838,8 @@ public void TestRename()
             Assert.Equal(expected, update.ToJson());
         }
 
-        // tests fail on .NET Core 2.1 with out of order key-value-pairs        
-#if !NETCOREAPP2_1
```

Comment:
An example of why negative `#if` are dangerous!

Reply:
I think the idea of this test (at least now) is to check behavior which supported only on older frameworks, so new frameworks not involved in this test

Reply:
I'm just pointing out that negative tests are dangerous because you don't know what the future holds.

Reply:
If the test is assuming ordered key/value pairs - which a hashtable or dictionary won't necessarily have - then we should fix the test rather than commenting it out for some targets.

Reply:
this was commented a long time, I think we have a ticket to fix it (which was discussed several times on grooming, and then we moved it into backlog as `Open`).

Reply:
so, I don't think that we need to work on it now